src/Antiforgery/src/RequireAntiforgeryTokenAttribute.cs

```diff
@@ -18,4 +20,16 @@ public class RequireAntiforgeryTokenAttribute(bool required = true) : Attribute,
     /// the validation check for the antiforgery token can be avoided.
     /// </remarks>
     public bool RequiresValidation { get; } = required;
+
+    /// <summary>
+    /// Gets or sets a value indicating the locations to look for the antiforgery token in the request.
+    /// If not set, <see cref="AntiforgeryOptions.RequestTokenSource"/> value used.
+    /// </summary>
+    public AntiforgeryRequestTokenSource RequestTokenSource
```

Comment:
Should we just make this nullable so anyone can determine the difference between unconfigured and explicitly `HeaderOrFormBody`? Then we can go back to just using auto properties.

Reply:
That was initial design, but such types are not supported by metadata